site/content/en/docs/Guides/Client SDKs/nodejs.md

```diff
@@ -22,9 +18,12 @@ Download the source {{< ghlink href="sdks/nodejs" >}}directly from GitHub{{< /gh
 Add the agones dependency to your project:
 
 ```sh
-$ npm install @googleforgames/agones
+$ npm install @google-cloud/agones-sdk
```

Comment:
Didn't feature shortcode wrap this, as to test, I pushed up a 1.3.0 npm image, so it works as of right now.